hack/update-generated-protobuf.sh

```diff
@@ -19,12 +19,36 @@ set -o nounset
 set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
-source "${KUBE_ROOT}/hack/lib/init.sh"
+source "${KUBE_ROOT}/build/common.sh"
 
 kube::golang::setup_env
 
-"${KUBE_ROOT}/hack/build-go.sh" cmd/libs/go2idl/go-to-protobuf cmd/libs/go2idl/go-to-protobuf/protoc-gen-gogo
+function prereqs() {
+  kube::log::status "Verifying Prerequisites...."
+  kube::build::ensure_docker_in_path || return 1
+  if kube::build::is_osx; then
+      kube::build::docker_available_on_osx || return 1
+  fi
+  kube::build::ensure_docker_daemon_connectivity || return 1
 
-"${KUBE_ROOT}/hack/after-build/update-generated-protobuf.sh" "$@"
+  KUBE_ROOT_HASH=$(kube::build::short_hash "$KUBE_ROOT/go-to-protobuf")
+  KUBE_BUILD_IMAGE_TAG="build-${KUBE_ROOT_HASH}"
+  KUBE_BUILD_IMAGE="${KUBE_BUILD_IMAGE_REPO}:${KUBE_BUILD_IMAGE_TAG}"
+  KUBE_BUILD_CONTAINER_NAME="kube-build-${KUBE_ROOT_HASH}"
+  KUBE_BUILD_DATA_CONTAINER_NAME="kube-build-data-${KUBE_ROOT_HASH}"
+  DOCKER_MOUNT_ARGS=(
```

Comment:
One problem with our docker builds (not your problem to fix here) is that builds don't work from a local env to a remote docker vm.  The volumes approach limits us - it would be better to do a pure docker build from the client than to do mounting.  I'll open a separate issue for that.